### ISSUE TITLE
HarmoniousBottleneck_2x

### ISSUE THREAD

**Issue by leonzgtee** [d-li14/HBONet#6]
开源代码中HarmoniousBottleneck_2x模块forward函数的elif self.stride == 2: return torch.cat((self.avgpool(x[:, -(self.oup - self.oup // 2):, :, :]), self.conv(x)), dim=1)，仅仅使用了cat，其他模块中包含了add之后再cat操作，文章结构示意图也是先add再cat，应该以哪个为准呢。

**Comment by d-li14** [comment]
We have tried torch.cat((self.avgpool(x[:, -(self.oup - self.oup // 2):, :, :]), self.avgpool(x[:, :(self.oup // 2), :, :]) + self.conv(x)), dim=1), which gives comparable or even inferior performance. So follow the code released.
